include/grpcpp/opencensus.h

```diff
@@ -42,6 +42,13 @@ void RegisterOpenCensusViewsForExport();
 // Returns the tracing Span for the current RPC.
 ::opencensus::trace::Span GetSpanFromServerContext(ServerContext* context);
 
+namespace experimental {
+// Setup gRPC's OpenCensus plugin to export monitoring and trace information to
+// stackdriver. Users of this API should not call `RegisterOpenCensusPlugin()`
+// or `RegisterOpenCensusViewsForExport()` in addition to this call.
+void RegisterOpenCensusPluginForGcpObservability();
```

Comment:
Is this what you will expose to the end users? In Java (and Go) we just have something like `grpcInit()` that users call which initializes everything for o11y. Will your `grpcInit` be implemented later?

Reply:
Moved to `GcpObservabiltyInit`